src/index.js

```diff
@@ -97,11 +99,13 @@ export default class extends Component {
       );
     }
 
-    const { title, source, onLoad, scrollEnabled } = this.props;
+    const { title, source, onLoad, scrollEnabled, style } = this.props;
     return createElement('iframe', {
       title,
       src: !source.method ? source.uri : undefined,
-      srcDoc: this.state.html || source.html,
+      srcDoc: this.injectedJavaScript(this.state.html || source.html),
+      width: style && style.width,
+      height: style && style.height,
```

Comment:
The width and props don't exist in the RN component. Why would you need them? Also this probably won't work if you generated your style with StyleSheet

Reply:
Yes, the width don't exist in the RN component [WebView.android.js](https://github.com/facebook/react-native/blob/master/Libraries/Components/WebView/WebView.android.js) , but our iframe need it :smile: , otherwise the width and height is some default value, ref to width and height in [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe)

Reply:
Fine with adding width and height ;)
Can you create a variable styleObj = StyleSheet.flatten(style), so that you can do styleObj.width and styleObj.height please? Because if your style is an array, this won't work!

Reply:
In new commit below, refactor to `width: '100%'` , thus no need to process style at all 😛